dbms/src/Databases/DatabaseMySQL.cpp

```diff
@@ -12,21 +14,33 @@
 #include <DataTypes/DataTypeNullable.h>
 #include <DataTypes/DataTypeFixedString.h>
 #include <Storages/StorageMySQL.h>
+#include <Parsers/ASTFunction.h>
 #include <Parsers/ParserCreateQuery.h>
 #include <Parsers/parseQuery.h>
 #include <Common/setThreadName.h>
+#include <Common/escapeForFileName.h>
+#include <Parsers/queryToString.h>
 #include <Parsers/ASTCreateQuery.h>
 #include <DataTypes/convertMySQLDataType.h>
 
+#include <Poco/File.h>
+#include <Poco/DirectoryIterator.h>
+
+
 
 namespace DB
 {
 
 namespace ErrorCodes
 {
     extern const int UNKNOWN_TABLE;
+    extern const int TABLE_IS_DROPPED;
+    extern const int TABLE_WAS_NOT_DROPPED;
+    extern const int TABLE_ALREADY_EXISTS;
+    extern const int UNEXPECTED_AST_STRUCTURE;
 }
 
+constexpr static const auto suffix = "_remove.flag";
```

Comment:
It will work correctly. But I recomment `.remove_flag` or `.remove.flag` just to avoid concatenation of table name with `_remove`.

Please compare:
```
items_to_remove_remove.flag
items_to_remove.remove_flag
```

Reply:
Done.